From ddf1c0f47e8d514ea92c5ad7eef3017935d83e46 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 20 Jun 2023 05:27:45 +0800
Subject: [PATCH 17/30] Revert "drm/amdgpu: fix an amdgpu_irq_put() issue in
 gmc_v9_0_hw_fini()"

This reverts commit eed63477ae065b69ee757e38a1cd48ec51de9719.
---
 drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c b/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
index 945cbdbc2f998..1673bf3bae55a 100644
--- a/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
@@ -1686,6 +1686,7 @@ static int gmc_v9_0_hw_fini(void *handle)
 		return 0;
 	}
 
+	amdgpu_irq_put(adev, &adev->gmc.ecc_irq, 0);
 	amdgpu_irq_put(adev, &adev->gmc.vm_fault, 0);
 
 	return 0;
-- 
2.25.1

